lib/pymedphys/_experimental/streamlit/apps/wlutz/_utilities.py

```diff
@@ -22,6 +23,17 @@
 from . import _dbf
 
 
+def filter_by(dataframe, column, value):
+    filtered = dataframe.loc[dataframe[column] == value]
+
+    return filtered
+
+
+# https://stackoverflow.com/a/16090640/3912576
+def natural_sort_key(s, _nsre=re.compile("([0-9]+)")):
```

Comment:
Consider import natsort instead of coding locally

Reply:
Yup, good call. I'll do that before merge.